### PR TITLE
Pylint updates, part four

### PR DESCRIPTION
**Pull request by ana**
... and last!

### CODE REVIEW

**Review by richtja — APPROVED** [review]
LGTM, thanks

File: avocado/utils/external/spark.py [hunk 1]
```diff
@@ -479,7 +479,7 @@ def add(self, input_set, item, i=None, predecessor=None, causal=None):
     def makeSet(self, token, sets, i):
         cur, next_item = sets[i], sets[i + 1]
 
-        ttype = token is not None and self.typestring(token) or None
+        ttype = token is not None and self.typestring(token) or None  # pylint: disable=R1709
```
Comment (beraldoleal) [inline avocado/utils/external/spark.py:482]: Hi @ana, my current pylint, is not raising any warning on this file. Could you please describe the current version you are using and why this is needed? Any CI using a modern pylint? Should we bump the pylint version requirement on requirements-dev.txt ?

File: requirements-dev.txt [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # inspektor (static and style checks)
 isort==5.8.0
 pyenchant==3.2.0
-pylint==2.8.3
+pylint==2.12.2
```
Comment (beraldoleal) [inline requirements-dev.txt:6]: Here is what I was looking for. :)

**Review by beraldoleal — APPROVED** [review]
LGTM, ignore my comment on initial commits. I just saw the last commit. Thanks.

**Pull request merged**